src/aliceVision/sensorDB/Datasheet.hpp

```diff
@@ -19,17 +20,17 @@ struct Datasheet
 {
   Datasheet() = default;
 
-  /**
+    /**
    * @brief Datasheet Constructor
    * @param brand
    * @param model
    * @param sensorSize
    */
-  Datasheet( const std::string& brand,
```

Comment:
IMHO, it's overkill to use std::move here. I would prefer consistency.
https://www.codesynthesis.com/~boris/blog/2012/07/03/efficient-argument-passing-cxx11-part3